code/game/objects/structures/signs/_signs.dm

```diff
@@ -4,21 +4,25 @@
 	opacity = 0
 	density = FALSE
 	layer = SIGN_LAYER
+	custom_materials = list(/datum/material/plastic = 2000)
 	max_integrity = 100
 	armor = list("melee" = 50, "bullet" = 0, "laser" = 0, "energy" = 0, "bomb" = 0, "bio" = 0, "rad" = 0, "fire" = 50, "acid" = 50)
-	var/buildable_sign = 1 //unwrenchable and modifiable
+	var/buildable_sign = TRUE //unwrenchable
 	rad_flags = RAD_PROTECT_CONTENTS | RAD_NO_CONTAMINATE
+	resistance_flags = FLAMMABLE
+	var/is_editable = FALSE //This determines if you can select this sign type when using a pen on a sign backing. False by default, set to true per sign subtype to override.
+	var/sign_change_name = "Sign - Blank" //sign_change_name is used to make nice looking, alphebetized and categorized names when you use a pen on a sign backing.
 
 /obj/structure/sign/basic
-	name = "blank sign"
-	desc = "How can signs be real if our eyes aren't real? Use a pen to change the decal."
```

Comment:
cant believe you did this great meme dirty like this

Reply:
Sacrifices must be made.

Reply:
no